docs/readmes/proposals/p011_intra_agw_mobility.md

```diff
@@ -0,0 +1,71 @@
+---
+id: p011_intra_agw_mobility
+title: Intra-AGW Mobility (S1 Mobility)
+hide_title: true
+---
+
+# Overview
+
+*Status: In-Review*\
+*Author: @shaddi*\
+*Feedback requested from: @ulaskozat, @pshelar*\
+*Last Updated: Mar 15, 2021*\
+*Targeted Release: 1.5*
+
+This document covers Intra-AGW mobility (aka S1 mobility support).
+
+## Goal
+
+Add S1 mobility support for intra-AGW mobility in situations where X2 handover is not available.
+
+## Introduction
+
+Currently, mobility in Magma is limited to RAN a single AGW where RAN elements support the X2 interface. This isn't always feasible; S1 mobility is the "fallback" method for mobility in LTE networks when X2 isn't avaialble. The scope of this proposal is enhancing intra-AGW mobility (that is, mobility that does not cross an AGW boundary) using S1 handover. Although *inter-AGW* mobility is not within scope of this proposal, the signalling mechanisms necessary for S1 handover will be useful for future mobility proposals.
+
+The intended use case for this proposal is single cell tower (or small cluster of cell sites) connected to a single AGW, potentially using a mix of eNB equipment, where the network operator wishes to support seamless mobility anchored at the AGW. UEs should maintain 
```

Comment:
The last sentence is incomplete. The first sentence can be rephrased as `The intended use case for this proposal is to support UE mobility across multiple eNBs with no X2 connections in between and anchored at the same AGW instance.